### ISSUE TITLE
#performance issue

### ISSUE THREAD

**Issue by Red025** [tensorflow/models#7698]
tensorflow/core/common_runtime/bfc_allocator.cc:211] Allocator (GPU_0_bfc) ran out of memory trying to allocate 1.54GiB. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory were available.

Object detection in video is running slow. Do anyone know how to increase the speed of the video?

**Comment by saberkun** [comment]
I also see this bfc_allocator.cc:211 error in other cases. It will not fail. This is probably related to the underlying library NCCL

**Comment by Red025** [comment]
Yes, it is running but the video is running slow. I am trying to run the object detection in video and the objects are detected but the video is running slow. What can i do to make the video to run fast?

**Comment by ravikyram** [comment]
@Red025 

Is this still an issue?.Please, let us know which pretrained model you are using and share related code .Thanks!

**Comment by Red025** [comment]
> @Red025
> 
> Is this still an issue?.Please, let us know which pretrained model you are using and share related code .Thanks!

https://github.com/EdjeElectronics/TensorFlow-Lite-Object-Detection-on-Android-and-Raspberry-Pi/blob/master/TFLite_detection_image.py

I have used this code along with the Google Coral USB Accelerator. Now it's working fine
